frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -20,11 +21,10 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
   };
   const requestSizeUnit = dropdownUnits.Ti;
   const [buttonDisabled, setButton] = React.useState(false);
-  const [requestSizeValue, setRequestSizeValue] = React.useState('1');
+  const [requestSizeValue, setRequestSizeValue] = React.useState('2');
```

Comment:
There are a few lines missing here now, if you look at https://github.com/openshift/console/pull/3252/files#diff-ecff7031320d48d9e801ba99a2fb7188R23

Will this work?

Reply:
Ya, 4.2 has units in `Ti` (that is without B), so those lines are not neccessary.